app.rb

```diff
@@ -3,11 +3,12 @@
 require_relative 'student'
 require_relative 'teacher'
 
-# Class containing app functionality for a basic UI
-class App
+# Class responsible for book management
+class BookManager
+  attr_accessor :books
+
```

Comment:
- You have done well so far, but your `app.rb` file is large and still needs refactoring. Please note that according to the project requirements, you are required to follow the [single-responsibility](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) principle. I suggest you extract some code into different class(es) and ensure they respect this single-responsibility principle. Please fix this 👍 👍